solargraph-rails.gemspec

```diff
@@ -1,4 +1,4 @@
-lib = File.expand_path('../lib', __FILE__)
+lib = File.expand_path('lib', __dir__)
```

Comment:
this is interesting, what is the importance of the difference here? (not pushing back, just curious)

Reply:
Hm, I think that's an autofix by Rubocop.